easyrsa3/easyrsa

```diff
@@ -689,14 +691,15 @@ $(display_dn req "$req_in")
 				print "nsComment = \"$EASYRSA_NS_COMMENT\""
 			case "$crt_type" in
 				server)	print "nsCertType = server" ;;
+				server-client)	print "nsCertType = server" ;;
 				client)	print "nsCertType = client" ;;
 				ca)	print "nsCertType = sslCA" ;;
 			esac
 		fi
 
 		# If type is server and no subjectAltName was requested,
 		# add one to the extensions file
-		if [ "$crt_type" = 'server' ];
+		if [ "$crt_type" = 'server' ] or [ "$crt_type" = 'server-client' ];
```

Comment:
The syntax here is odd.  This should be:

`if [ "$crt_Type" = "server" -o "$crt_type" = "server-client" ];`